lib/src/components/buttons/details_button.dart

```diff
@@ -13,12 +13,33 @@ class DetailsButton extends StatelessWidget {
   /// It's a variable that stores the title of the button.
   final String title;
 
+  /// fontSize for title
+  final double fontSize;
+
+  /// letterSpacing for title
+  final double letterSpacing;
+
+  /// how much free space should be between title and chevron
+  final double spacingToChevron;
+
+  /// how much free space should be after chevron
+  final double spacingAfterChevron;
+
+  static const _defaultFontSize = 19.2;
+  static const _defaultLetterSpacing = -0.49;
+  static const _defaultSpacingToChevron = 2.0;
+  static const _defaultSpacingAfterChevron = 0.0;
+
   ///Constructor
   const DetailsButton({
     Key? key,
     required this.onTap,
     required this.color,
     required this.title,
+    this.fontSize = _defaultFontSize,
+    this.letterSpacing = _defaultLetterSpacing,
+    this.spacingToChevron = _defaultSpacingToChevron,
+    this.spacingAfterChevron = _defaultSpacingAfterChevron,
```

Comment:
The constructor seems to have a bit too many parameters here -- let's see if we can refactor it.
IMO we could either go with:

1. the suggestion above (combining the fields into `textStyle` and `chevronPadding`)
2. actually make the children of this Widget fully customizable by adding a `child` parameter and making the Text + chevron external.

I'm more inclined towards the second option, as it gives way more flexibility -- we might need it for variations 3 and 4.